### PR TITLE
add lesson02 fizz_buzz file

### PR DESCRIPTION
**Pull request by zzsocool**
I am sorry I accidently pushed two folders
I am not sure how to delete from my forked repo before the pull request.
Please ignore or delete the folder " students/Shen_ZF/lesson02/Fizz_Buzz.py"
Thank you.

### CODE REVIEW

**Comment by zzsocool** [comment]
add Grib_printer file

**Comment by zzsocool** [comment]
add series file

File: students/Z_shen/lesson02/Fizz_Buzz.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+L= range(1,101)
+
+for i in L:
```
Comment (natasha-aleksandrova) [inline students/Z_shen/lesson02/Fizz_Buzz.py:3]: variable names should be lower case, and preferably more readable names.

**Review by natasha-aleksandrova — APPROVED** [review]
code looks good 👍

**Pull request merged**